src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import core.basesyntax.suppliers.FigureSupplier;
+
+public class Main extends FigureSupplier {
+    public static void main(String[] args) {
+        Figure[] figures = new Figure[5];
```

Comment:
5 is magic number
https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names